src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1059,6 +1059,14 @@ void testPotionEffects()
 
 	}
 
+	@Test
+	void clearPotionEffects()
+	{
+		PotionEffect effect = new PotionEffect(PotionEffectType.CONFUSION, 5, 1);
+		player.addPotionEffect(effect);
+		assertTrue(player.clearActivePotionEffects());
+	}
+
```

Comment:
Is there a reason you are testing this on PlayerMockTest and not LivingEntityMockTest

Reply:
PlayerMockTest has all potion effect tests that should probably be in LivingEntityMockTest - should all of these be refactored?

Reply:
yes